pkg/alertmanager/multitenant.go

```diff
@@ -401,6 +402,10 @@ func createMultitenantAlertmanager(cfg *MultitenantAlertmanagerConfig, fallbackC
 			Name: "cortex_alertmanager_tenants_owned",
 			Help: "Current number of tenants owned by the Alertmanager instance.",
 		}),
+		configSize: promauto.With(registerer).NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cortex_alertmanager_config_size_bytes",
+			Help: "Size of the last received alertmanager configuration.",
+		}, []string{"user"}),
```

Comment:
The name is a bit ambiguous, it's not clear that we're only using this for Grafana configurations. Maybe something like `cortex_alertmanager_grafana_config_size_bytes` is enough to avoid confusions.

The [`multitenantAlertmanagerMetrics` struct](https://github.com/grafana/mimir/blob/b810cbe07175145c9c63d046df58637916ac9a46/pkg/alertmanager/multitenant.go#L185-L188) might be a better place for defining this metric. We could add a new field (e.g. `grafanaConfigSize`) and initialize the gauge in [`newMultitenantAlertmanagerMetrics()`](https://github.com/grafana/mimir/blob/b810cbe07175145c9c63d046df58637916ac9a46/pkg/alertmanager/multitenant.go#L190-L206)

Reply:
> The name is a bit ambiguous, it's not clear that we're only using this for Grafana configurations. Maybe something like cortex_alertmanager_grafana_config_size_bytes is enough to avoid confusions.

Yep, good point - changed now 👍 

> The [multitenantAlertmanagerMetrics struct](https://github.com/grafana/mimir/blob/b810cbe07175145c9c63d046df58637916ac9a46/pkg/alertmanager/multitenant.go#L185-L188) might be a better place for defining this metric. We could add a new field (e.g. grafanaConfigSize) and initialize the gauge in [newMultitenantAlertmanagerMetrics()](https://github.com/grafana/mimir/blob/b810cbe07175145c9c63d046df58637916ac9a46/pkg/alertmanager/multitenant.go#L190-L206)

Agreed. Also fixed now 👍